pages/admin/users.tsx

```diff
@@ -112,8 +125,14 @@ export default function UserPage() {
       } else {
         setFilteredUsers([...users]);
       }
+      // if user permission is admin, filter to hackers only
+      if (user && user.permissions[0] === 'admin') {
+        setFilteredUsers([...users.filter((user) => user.user.permissions.includes('hacker'))]);
```

Comment:
yea but if i do that then it would filter for admins and super_admins as it includes admins. or i could do this if u want `if (user && user.permissions.includes('admin') && !user.permissions.includes('super_admin'))`

Reply:
Verify that. Trying this in a node environment says otherwise.
```typescript
let x = ["hacker", "super_admin"]
x.includes("admin") // returns false
x.includes("super_admin") // returns true
```

Reply:
got it